client/connector_create.go

```diff
@@ -43,6 +43,9 @@ func generateConnectorName(namespace string, cli kubernetes.Interface) string {
 }
 
 func (cli *VanClient) ConnectorCreateFromFile(ctx context.Context, secretFile string, options types.ConnectorCreateOptions) (*corev1.Secret, error) {
+	if secretFile == cli.SecretFile {
```

Comment:
I don't see how this can work in general. E.g. when used from the cli, the process exists when the token is created so the inmemory state is lost. Also there could be many connection tokens generated.

Reply:
Ah!  You mean 'exits', right?
Cripes.  I didn't think of the CLI.
OK.... back to drawingboard.

Reply:
Doh, yes I meant 'exits'!